collectors/python.d.plugin/uwsgi/uwsgi.chart.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# Description: uwsgi netdata python.d module
+# Author: Robbert Segeren (robbert-ef)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+from copy import deepcopy
+from bases.FrameworkServices.SocketService import SocketService
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
+priority = 60000
+retries = 60
+
+ORDER = ['requests', 'tx', 'avg_rt', 'memory_rss', 'memory_vsz', 'exceptions', 'harakiri', 'respawn', ]
+
+# NOTE: lines are created dynamically in `check()` method
+CHARTS = {
+    'requests': {
+        'options': [None, 'Requests', 'requests/s', 'requests', 'uwsgi.requests', 'stacked'],
+        'lines': [
+            ['requests.%s', 'worker %s', 'incremental']
+        ]},
+    'tx': {
+        'options': [None, 'Transmitted data', 'KB/s', 'requests', 'uwsgi.tx', 'stacked'],
+        'lines': [
+            ['tx.%s', 'worker %s', 'incremental']
+        ]},
+    'avg_rt': {
+        'options': [None, 'Average request time', 'ms', 'requests', 'uwsgi.avg_rt', 'line'],
+        'lines': [
+            ['avg_rt.%s', 'worker %s', 'absolute']
+        ]},
+    'memory_rss': {
+        'options': [None, 'RSS (Resident Set Size)', 'MB', 'memory', 'uwsgi.memory_rss', 'stacked'],
+        'lines': [
+            ['memory_rss.%s', 'worker %s', 'absolute', 1, 1024 * 1024]
+        ]},
+    'memory_vsz': {
+        'options': [None, 'VSZ (Virtual Memory Size)', 'MB', 'memory', 'uwsgi.memory_vsz', 'stacked'],
+        'lines': [
+            ['memory_vsz.%s', 'worker %s', 'absolute', 1, 1024 * 1024]
+        ]},
+    'exceptions': {
+        'options': [None, 'Exceptions', 'exceptions', 'exceptions', 'uwsgi.exceptions', 'line'],
+        'lines': [
+            ['exceptions.%s', 'worker %s', 'incremental']
+        ]},
+    'harakiri': {
+        'options': [None, 'Harakiris', 'harakiris', 'harakiris', 'uwsgi.harakiris', 'line'],
+        'lines': [
+            ['harakiri_count.%s', 'worker %s', 'incremental']
```

Comment:
I am not sure about `worker` prefix.  Do we really need it? (for every chart for evey dim)

Reply:
I've added this prefix so it more clear what the chart contains, without it, dimensions would just be `1`, `2`:

![image](https://user-images.githubusercontent.com/22441832/47017401-6e214e00-d152-11e8-847e-9c9db616e3b8.png)

I'd rather have the `pid` there, but that can change over time in case of respawn and I dont know how to change dimensions run-time.